conda_forge_tick/migrators/version.py

```diff
@@ -622,6 +632,22 @@ def pr_body(self, feedstock_ctx: FeedstockContext) -> str:
             )
         for p in pred:
             body += template.format(name=p[0], new_version=p[1])
+
+        update_deps = feedstock_ctx.attrs.get("conda-forge.yml", {}).get("bot", {}).get("update_deps", False)
+        if update_deps == 'hint':
+            deps = extract_deps_from_source(os.path.join(feedstock_dir, 'recipe'), self.import_cf_map)
+            dep_comparison = compare_depfinder_audit(deps, feedstock_ctx.attrs, feedstock_ctx.attrs['name'])
+            if dep_comparison:
+                df_cf = "\n-".join(self.dep_comparison["df_minus_cf"])
+                cf_df = "\n-".join(self.dep_comparison["cf_minus_df"])
+                hint = f'\n\nDependency Analysis\n-------------' \
+                       f'Analysis of the source code by depfinder seems to show a discrepency between' \
```

Comment:
maybe make `depfinder` a link so ppl can find it and look at it?

Reply:
Actually @scopatz brought up the point that we might use multiple systems to provide inspection at some point, so we might want to leave this out.

Reply:
sgtm!